tests/network/autofs_test_autofs_client.pm

```diff
@@ -0,0 +1,38 @@
+# SUSE's openQA tests
```

Comment:
I think it is not necessary to have test in filenames autofs_test_autofs_client and autofs_test_nfs_server.

Reply:
`autofs_client` is good enough

Reply:
new names are: `autofs_client`and `autofs_server`